sequencer/src/context.rs

```diff
@@ -25,13 +30,20 @@ pub struct SequencerContext<TYPES: NodeType, I: NodeImplementation<TYPES>> {
     state_signatures: Arc<RwLock<StateSignatureMemStorage>>,
 }
 
-impl<TYPES: NodeType, I: NodeImplementation<TYPES>> SequencerContext<TYPES, I> {
+impl<N: network::Type> Clone for SequencerContext<N> {
```

Comment:
I assume this is implemented manually because the derived implementation requires `N: Clone`? You can use `Derivative` to get around this:

```rust
#[derive(Derivative)]
#[derivative(Clone(bound = ""))]
```

Reply:
Yes `Derivative` works! [ddd5882](https://github.com/EspressoSystems/espresso-sequencer/pull/879/commits/ddd588239450fbc6ff311db35354315a475272f0)